src/cpu/o3/store_set.hh

```diff
@@ -62,20 +63,35 @@ struct ltseqnum
 class StoreSet
 {
   public:
-    typedef unsigned SSID;
+    typedef Addr SSID;
+    typedef replacement_policy::Base BaseReplacementPolicy;
```

Comment:
Remove this typedef, as it costs 61 characters to add it, but it currently (after my requested changes) saves 4*3=12 characters. One would have to use this defined type quite a lot of times to be worth it.